envs/monkey_zoo/terraform/firewalls.tf

```
@@ -272,3 +272,90 @@ resource "google_compute_firewall" "allow-all-tunneling2" {
   // Here goes your public IP so you can SSH/RDP in the instances
   source_ranges = ["127.0.0.1/32"]
 }
+
+resource "google_compute_firewall" "allow-rdp64-from-island" {
+  name    = "allow-rdp64-from-island"
+  network = google_compute_network.monkeyzoo.name
+
+  allow {
+    protocol = "all"
+  }
+  priority = "999"
+  source_tags = ["island"]
+  target_tags = ["rdp-64"]
+}
```

Can these rules be consolidated?
```suggestion
resource "google_compute_firewall" "allow-rdp64-and-island" {
  name    = "allow-rdp64-and-island"
  network = google_compute_network.monkeyzoo.name

  allow {
    protocol = "all"
  }
  priority = "999"
  source_tags = ["island", "rdp64"]
  target_tags = ["rdp-64", "island"]
}
```

---

We can. Thanks!